packages/components/organisms/f-cookie-banner-static/package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "name": "@justeat/f-cookie-banner-static",
+  "private": true,
+  "version": "1.0.0",
```

Comment:
Had to add these fields so that Lerna would manage the dependencies. Given `private:true` is set, there's no risk of this accidentally getting published to NPM.

As mentioned in the PR description, static files will continue to be published in `f-cookie-banner/dist/static` as part of a publish to `f-cookie-banner`